Wbooks/Models/Book.swift

```diff
@@ -8,10 +8,30 @@
 import Foundation
 import UIKit
 
-struct Book {
+struct Book: Codable {
     
     // MARK: Properties
-    let cover: UIImage
-    let title, author: String
+    let id: Int
+    let title, author, genre, year, image: String
+    
+    enum BookKey: String, CodingKey {
+        case id = "id"
+        case title = "title"
+        case author = "author"
+        case genre = "genre"
+        case year = "year"
+        case image = "image"
+    }
+    
+    // MARK: Inizialization
+    public init(from: Decoder) {
+        let container = try! from.container(keyedBy: BookKey.self)
+        id = try! container.decode(Int.self, forKey: .id)
+        title = try! container.decode(String.self, forKey: .title)
+        author = try! container.decode(String.self, forKey: .author)
+        genre = try! container.decode(String.self, forKey: .genre)
+        year = try! container.decode(String.self, forKey: .year)
+        image = try! container.decode(String.self, forKey: .image)
+    }
```

Comment:
I think this is not necessary, since the keys are named like the properties by default.